.github/workflows/linux.yml

```diff
@@ -0,0 +1,25 @@
+name: Linux desktop CI
+
+on:
+  pull_request: 
+    types: [synchronize, review_requested, ready_for_review]
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-building:
```

Comment:
Is `test-building` a reserved `github` keyword or the name of a job dedicated to building the tests executable or is it a name you have chosen?

Reply:
`test-building` was a name I picked, as the job was essentially just a test to see if the build was still working.
I see how this wording can be confusing, especially with future unit tests coming, and updated the name